sii/resource.py

```diff
@@ -231,10 +231,20 @@ def get_factura_recibida_dict(invoice, rectificativa=False):
     return obj
 
 
+def refactor_nifs(invoice):
+    if invoice.partner_id.vat.lower().startswith('es'):
+        invoice.partner_id.vat = invoice.partner_id.vat[2:]
+    if invoice.company_id.partner_id.vat.lower().startswith('es'):
+        invoice.company_id.partner_id.vat = \
+            invoice.company_id.partner_id.vat[2:]
```

Comment:
This can be done with:

```python
import re

invoice.partner_id.vat = re.sub('^ES', '', invoice.partner_id.vat)
invoice.company_id.partner_id.vat = re.sub('^ES', '', invoice.company_id.partner_id.vat)